inst/odin/model-targeted-vax.R

```diff
@@ -167,186 +161,165 @@ initial(prioritisation_step_1st_dose_children) <- 1
 initial(prioritisation_step_1st_dose_adults) <- 1
 initial(prioritisation_step_2nd_dose_adults) <- 1
 
-## now that we know the step we are on, we know who is eligible per age group to
+## now that we know the step we are on, we know who is eligible per age group to 
 ## be vaccinated
 
+## create indicator for which groups are allowed to be vaccinated based on:
+## 1) if they are an adult or a child (via is_child)
+## 2) if they are in a prioritised group, dependent on being adult or child (prioritisation_strategy_X)
+## 3) if the target for this group has not yet been met (target_met_X_t)
+
+dim(give_dose1_children) <- c(n_group)
+give_dose1_children[] <- 
+  (is_child[i] * ceiling(prioritisation_strategy_children[
+    i, prioritisation_step_1st_dose_children]) * (1 - target_met_children_t[i])) 
```

Comment:
```suggestion
  is_child[i] * coverage_target_1st_dose_children[i] * (1 - target_met_children_t[i]))
```
I think this is defined above so can use the object here to make more readable - I've suggested the equivalent amendments below if you agree